plugins/nordic-hid/README.md

```diff
@@ -31,7 +31,22 @@ This plugin supports the following protocol ID:
 ## GUID Generation
 
 For GUID generation the standard HIDRAW DeviceInstanceId values are used
-with the addition of the target board and bootloader name:
+with the addition of the target board name, bootloader name and generation.
+The generation string is an application-specific property that allows to distinguish configurations
+that use the same board and bootloader, but are not interoperable.
+
+GUID examples:
+
+* `HIDRAW\VEN_1915&DEV_52DE&BOARD_nrf52840dk&BL_B0&GEN_default` -> b76d19e5-d745-5c0b-b870-b1b6d78e3c63
+* `HIDRAW\VEN_1915&DEV_52DE&BOARD_nrf52840dk&BL_MCUBOOT&GEN_office` -> 1728d5e4-e535-57f9-addc-a6c3765f81db
+
+The generation parameter is not supported by older versions of fwupd.
+Handling this parameter was introduced later.
+For compatibility with firmware updates released before support of the generation parameter
+was intoduced, devices with "default" generation report and additional GUID generated without
```

Comment:
```suggestion
To ensure compatibility with firmware updates that were released before introducing the support for the generation parameter,
devices with the "default" generation report and additional GUID were generated without
```

Reply:
Should be `...an additional GUID... `. Sorry for my typo

Reply:
Rephrased